bot.py

```diff
@@ -332,6 +361,56 @@ async def listen_for_response(
 discord.utils.setup_logging(handler=handler)
 
 
+@bot.command(
+    name=SYNC_COMMAND_NAME,
+    description="Syncs command list. Any new commands will be available for use.",
+    help="The command has an optional second argument to state whether to sync all or only guild commands. Please specify it by either mentioning {}.",
+)
+@commands.check(is_staff_from_ctx)
+async def sync(ctx: commands.Context, sync_type: Literal["all", "guild"] = "all"):
+    """
+    Syncs and registers and new commands with Discord. This command is a
+    top-level command to prevent disabled cogs from disabling sync
+    functionality.
+    """
+    match sync_type:
+        case "all":
+            only_guild_commands = False
+        case "guild":
+            only_guild_commands = True
+    async with ctx.typing(ephemeral=True):
+        global_cmds_synced, guild_cmds_synced = await bot.sync_commands(
+            only_guild_commands,
+        )
+        res_msg = (
+            f"{len(global_cmds_synced)} global commands were synced."
+            if not only_guild_commands
+            else ""
+        )
+        for guild_id, cmds in guild_cmds_synced.items():
+            guild_info = bot.get_guild(guild_id)
+            guild_name_id = f"guild with id {guild_id}"
+            if guild_info:
+                guild_name_id = f'"{guild_info.name}" (id: {guild_id})'
+            res_msg += f"\n{len(cmds)} guild commands were synced in {guild_name_id}."
+        res_msg = res_msg.strip("\n")
+        await ctx.send(res_msg)
+
+
+@sync.error
+async def sync_error(ctx: commands.Context, err):
+    logging.error(err)
+    if isinstance(err, commands.BadLiteralArgument):
+        literals = list(err.literals)
+        literals[-1] = f"or {literals[-1]}"
+        for literal in literals:
+            literal = f'"{literal}"'
+        assert sync.help is not None
+        msg = sync.help.format((", " if len(literals) > 2 else " ").join(literals))
```

Comment:
`", ".join(literals)`?

Reply:
Constructs the parallel sentence by joining the elements with `", "` (e.g. `", ".join(['"all"', '"guild"', 'or "guild-only"']) == '"all", "guild", or "guild-only"'`).